Dockerfile

```diff
@@ -0,0 +1,42 @@
+FROM quay.io/ukhomeofficedigital/alpine:v3.14 as builder
+
+USER root
+
+RUN apk add --no-cache openjdk11-jre
+
+COPY build/libs/*.jar .
+
+RUN java -Djarmode=layertools -jar *.jar extract
```

Comment:
I'd consider moving the whitespace about: everything being double-spaced doesn't help readability; group related parts together like in the latter part of this Dockerfile.

Reply:
Good shout I've grouped it up a bit more readably.